Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -260,6 +260,17 @@ - (void)refreshTagEditorOffsetWithCoordinator:(id<UIViewControllerTransitionCoor
     } completion:nil];
 }
 
+- (void)refreshSearchHighlightIfNeededWithCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    if (!self.searching) {
+        return;
+    }
+
+    [coordinator animateAlongsideTransition:nil completion:^(id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
+        [self refershSearchHighlight];
+    }];
```

Comment:
Right now you can see the highlight jumping from one position to another because we refresh it in `completion` block. What we can do is refresh it in `animation` block, like this:
```
[coordinator animateAlongsideTransitionInView:nil animation:^(id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
            [self refershSearchHighlight];
    } completion:nil];
```

Reply:
nice worked like a charm!